ansible/www-standalone/resources/config/nodejs.org

```diff
@@ -34,6 +34,10 @@ server {
         
         try_files $uri $uri/ =404;
     }
+    
+    location = /en/404.html {
+        add_header Cache-Control "no-cache" always;
+    }
```

Comment:
maybe you could have kept the `@name` and used `alias /en/404.html;` ?

Reply:
That would also work.

Reply:
Issue is `alias` would require full path